### ISSUE TITLE
Led "w" to a "nan" when change "stop" param of "trX" in 01_linear_regression

### ISSUE THREAD

**Issue by huang-xu** [nlintz/TensorFlow-Tutorials#67]
In 01_linear_regression.ipyn, when change the "stop" parameter of `np.linspace` in line `trX = np.linspace(-1, 1, 101)` to a larger number like 20 (i.e. `trX = np.linspace(-1, 20, 101)`), then the learned weight  `w` will be `nan`.

**Comment by akashgoel1197** [comment]
sir you have to change the learning rate to 0.00001 for trX = np.linspace(-1, 20, 101) , then the learned weight will not be nan.
